Formula/linode-cli.rb

```diff
@@ -0,0 +1,87 @@
+class LinodeCli < Formula
+  include Language::Python::Virtualenv
+
+  desc "CLI for the Linode API"
+  homepage "https://www.linode.com/products/cli/"
+  url "https://github.com/linode/linode-cli/archive/refs/tags/5.14.0.tar.gz"
+  sha256 "64ec447910eda75759e88e814846432b75ff08211ff1d1a86b223d5db5cd9902"
+  license "BSD-3-Clause"
+  head "https://github.com/linode/linode-cli.git", branch: "master"
+
+  depends_on "rust" => :build # for cryptography
```

Comment:
You don't have a `cryptography` resource though. Does it need `cryptography` because it's trying to install `poetry`? If so, let's just depend on `poetry` directly and use that to build whatever it is it wants to build with `poetry`.

See other formulae that have a `poetry` dependency for examples.

Reply:
Thanks for the tip. Got it done.